javascript/javascript2/week2/movies.js

```diff
@@ -0,0 +1,250 @@
+    {"title": "88 Minutes","year": 2007,"rating": 5.9,"votes": 69067,"running_times": 6480},
+    {"title": "8MM","year": 1999,"rating": 6.5,"votes": 110165,"running_times": 7380},
+    {"title": "8½","year": 1963,"rating": 8.1,"votes": 88077,"running_times": 8280},
+    {"title": "9","year": 2009,"rating": 7.1,"votes": 118743,"running_times": 4740},
+    {"title": "9 Songs","year": 2004,"rating": 4.9,"votes": 19175,"running_times": 4260},
+    {"title": "99 Homes","year": 2014,"rating": 7.1,"votes": 23100,"running_times": 6720},
+    {"title": "A Beautiful Mind","year": 2001,"rating": 8.2,"votes": 692444,"running_times": 8100},
+    {"title": "A Better Life","year": 2011,"rating": 7.2,"votes": 13931,"running_times": 5880},
+    {"title": "A Bigger Splash","year": 2015,"rating": 6.4,"votes": 17641,"running_times": 7500},
+    {"title": "A Boy and His Dog","year": 1975,"rating": 6.6,"votes": 14010,"running_times": 5460},
+    {"title": "A Bridge Too Far","year": 1977,"rating": 7.4,"votes": 43717,"running_times": 10500},
+    {"title": "A Bronx Tale","year": 1993,"rating": 7.8,"votes": 107443,"running_times": 7260},
+    {"title": "A Bug's Life","year": 1998,"rating": 7.2,"votes": 229465,"running_times": 5700},
+    {"title": "A Christmas Carol","year": 2009,"rating": 6.8,"votes": 82681,"running_times": 5760},
+    {"title": "A Christmas Story","year": 1983,"rating": 8,"votes": 113032,"running_times": 5640},
+    {"title": "A Cinderella Story","year": 2004,"rating": 5.9,"votes": 69295,"running_times": 5700},
+    {"title": "A Civil Action","year": 1998,"rating": 6.5,"votes": 24558,"running_times": 6900},
+    {"title": "A Clockwork Orange","year": 1971,"rating": 8.3,"votes": 623645,"running_times": 8160},
+    {"title": "A Close Shave","year": 1995,"rating": 8.2,"votes": 33053,"running_times": 1800},
+    {"title": "A Cock and Bull Story","year": 2005,"rating": 6.8,"votes": 11624,"running_times": 5640},
+    {"title": "A Cure for Wellness","year": 2016,"rating": 6.4,"votes": 50746,"running_times": 8760},
+    {"title": "A Dangerous Method","year": 2011,"rating": 6.5,"votes": 87773,"running_times": 5940},
+    {"title": "A Day at the Races","year": 1937,"rating": 7.7,"votes": 11369,"running_times": 6660},
+    {"title": "A Dirty Shame","year": 2004,"rating": 5.1,"votes": 11191,"running_times": 5340},
+    {"title": "A Dog's Purpose","year": 2017,"rating": 6.9,"votes": 38675,"running_times": 6000},
+    {"title": "A Face in the Crowd","year": 1957,"rating": 8.2,"votes": 10910,"running_times": 7560},
+    {"title": "hello world hello","year": 1957,"rating": 8.2,"votes": 10910,"running_times": 7560},
+    {"title": "A Fantastic Fear of Everything","year": 2012,"rating": 5.9,"votes": 12446,"running_times": 6000}
+];
+//short and log titles
+const shortTiteles =  movies.filter(movie=> movie.title.length<10).map(movie=>movie.title);
+moviesShortTitles.push(shortTiteles)
+console.log(moviesShortTitles);
```

Comment:
You don't need to create another array, as the filter function itself will generate an array. That means in here shortTiteles is an array created after filtering the movies. So no need for 'moviesShortTitles'

Reply:
yes make sense  :)